content/resources/registry/publishing/_index.md

```diff
@@ -0,0 +1,183 @@
+
+#### Step 3. Add test and training data
+
+#### Step 4. Add code and models
+If you want to publish __code__ with the recognizer, add it to the `src` folder.
+
+Similarly, if you have __a trained model or other artefacts__ produced while developing your recognizer,
+put them to `artifacts` folder.
+
+#### Step 5. Publish your recognizer in GitHub
+Once you are ready to publish your recognizer you can do so by setting your repository public. 
+1. In your repository go to the Settings tab
+![Picture of GitHub settings tab](github-settings-tab.png)
+2. In the _General_ section, scroll to the bottom "Danger Zone"
+3. Click the button _Change visibility_ and select _Change to public_
+![Screenshot of Visibility settings](github-change-visibility.png)
+
+#### Step 6. Generate a DOI using Zenodo
+
+Generating a DOI will make this repository [Findable and Citable](https://ardc.edu.au/resource/fair-data/),
+enabling others in the scientific community reference and build upon your work.
+
+1. Go to [Zenodo](https://zenodo.org/login).
+2. Use your GitHub account to log in.
+3. Once you are logged in, choose "Github" from the drop-down menu by clicking the arrow next to your email address
+in the top right-hand side menu. 
+![Picture of Zenodo drop down menu](zenodo-github-page.png)
+4. On this page you will see a list of your repositories. Flip the switch next to the recognizer repository you
+created to ON-position.
+![Picture of Zenodo repository switch](zenodo-repo-switch.png)
+5. Go to Github and [create a release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository).
+Zenodo will automatically generate a DOI for this release which you should now see in Zenodo GitHub page.
+Zenodo will generate a new DOI version for each release.
+[Read about Zenodo Versioning](https://help.zenodo.org/#versioning).
+6. Click either the title or the DOI-badge to go to your generated DOI-page. 
+7. Finally, you can copy the DOI-badge from the DOI-page by clicking the DOI-badge on the side panel.
+![Picture of Zenodo DOI-badge in the side panel](zenodo-doi-badge.png)
+We recommend copying the Markdown text and adding it to your README-file in your GitHub Recocnizer Repository.
+
+#### Step 7. Add or modify a citation information file
+
+To make your GitHub recognizer repository easily citable we recommend adding a citation information file (`CITATION.cff`).  We have added a citation file to the recognizer template for your convenience. 
+
+1. Modify the `CITATION.cff` file in your repository.
+![Screenshot of the Citation.cff file.](citation-file-edit.png)
+You can edit files in your browser by clicking the pen icon.
+Replace the author name, the ORCID, the title and the github link fields with your own details.
+2. Alternatively you can use the
+[CITATION.cff generation website](https://citation-file-format.github.io/cff-initializer-javascript/#/) to create the
```

Comment:
let's simplify to just one path. I'd vote for go to cffinit, then create the record, and the replace the contents of the cff file in your repo with the one generated